shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java

```diff
@@ -63,8 +63,7 @@ protected Mono<ClientResponse> doRequest(final ServerWebExchange exchange, final
                         .flatMap(bytes -> Mono.fromCallable(() -> Optional.ofNullable(bytes))).defaultIfEmpty(Optional.empty())
                         .flatMap(option -> {
                             final ClientResponse.Builder builder = ClientResponse.create(response.statusCode())
-                                    .headers(headers -> headers.addAll(response.headers().asHttpHeaders()))
-                                    .cookies(cookies -> cookies.addAll(response.cookies()));
```

Comment:
why remove cookies setting？

Reply:
Setting cookies is just part of setting headers. So the previous version codes may lead setting cookies twice.

Reply:
Is there a time when cookies is not passed from the header?

Reply:
the data of `response.cookies()` is the subset of `response.headers().asHttpHeaders()`, therefore setting headers is also setting cookies in the sam time. I have never met the case that cookies is not present in headers. Is this case exists?